packages/documentation/src/shared/nb-bootstrap/ngb-component-demo-link.mdx

```diff
@@ -0,0 +1,8 @@
+<a
+  className="btn btn-rg btn-secondary btn-animated"
+  href={`https://design-system.post.ch/#/ng-bootstrap-samples/${props.component}`}
```

Comment:
This link will change to something like https://**demo**.design-system.post.ch/#/bla, but we'll update that in a separate step when the environment is set up.